include/mbedtls/mbedtls_config.h

```diff
@@ -328,6 +328,7 @@
 //#define MBEDTLS_SHA1_ALT
 //#define MBEDTLS_SHA256_ALT
 //#define MBEDTLS_SHA512_ALT
+//#define MBEDTLS_SHA3_ALT
```

Comment:
See @tom-cosgrove-arm comments on #5820, where new implementations shouldn't use the "ALT" mechanism.